scripts/cocoapods/utils.rb

```diff
@@ -93,6 +93,18 @@ def self.fix_library_search_paths(installer)
         end
     end
 
+    def self.fix_react_bridging_header_search_paths(installer)
+        installer.target_installation_results.pod_target_installation_results
+            .each do |pod_name, target_installation_result|
+                target_installation_result.native_target.build_configurations.each do |config|
+                    # For third party modules who have React-bridging dependency to search correct headers
+                    config.build_settings['HEADER_SEARCH_PATHS'] ||= '$(inherited) '
+                    config.build_settings['HEADER_SEARCH_PATHS'] << '"$(PODS_ROOT)/Headers/Private/React-bridging/react/bridging" '
+                    config.build_settings['HEADER_SEARCH_PATHS'] << '"$(PODS_CONFIGURATION_BUILD_DIR)/React-bridging/react_bridging.framework/Headers" '
+            end
+        end
+    end
+
```

Comment:
@Kudo, thanks for all your hard work with this. I'm curious why this is needed on main. Did we not fix the `.podspec` files? This looks wrong since it adds private headers to the search path among other paths that should've been added by CocoaPods if the `.podspec` was fixed.

Reply:
It seems like [React-bridging.podspec](https://github.com/facebook/react-native/blob/main/ReactCommon/React-bridging.podspec) does not have `public_header_files` set. Would that have fixed the underlying issue?

Reply:
@tido64 this is for third party libraries integration, e.g. react-native-vision-camera. originally, [my workaround](https://github.com/facebook/react-native/commit/f97c6a5b498eec95e99a02c7842cb2ae160cd6cd) was for React-bridging errors in use_frameworks mode and i added the header search paths to ReactCommon. it turns out some third party modules have dependencies to ReactCommon or React would also need the header search paths. that's why i am proposing to add them from CocoaPods post_install.

these are all workarounds 😔 in the future, we should remove the nested header imports, e.g. `#include <react/bridging/Bridging.h>` to support use_frameworks mode.